docs/backend/guides/how-to/manage-golang-with-asdf.md

```diff
@@ -46,7 +54,7 @@ Then run `asdf global golang 1.15.10`
 
 ### Reload your shell
 
-Reload your shell, open a new shell or source your rc file.
+Reload your shell, open a new shell or source your resource file.
```

Comment:
I've generally seen the terminology rc file or .rc file. Personally I would keep this as rc file and add resource in parenthesis. While resource is probably more accurate since you can source your bash_profile, I'd still advocate for rc. Fwiw it also doesn't look like there is a consensus on what rc actually stands for. I had "learned"  it was run command/control (can't recall which, but I see reference to both) in the context of unix but also see that resource is also a common definition.

Reply:
Looks like **Run Commands** would be a better name here. So we can keep them as `rc` file, or better yet name the files explicitly rather than have them be "rc files" since this would be an actual file that people edit, `bashrc`, `zshrc` or `fishrc`.

Reply:
Honestly though, on second thought I don't see a big value in keeping these docs around for long. The official `asdf` documentation is going to be kept more up-to-date than whatever we run.